platform/viewer/src/appExtensions/GenericViewerCommands/commandsModule.js

```diff
@@ -1,5 +1,8 @@
-import { redux, utils } from '@ohif/core';
+import { redux } from '@ohif/core';
 import store from './../../store';
+
+import { commandsManager } from './../../App.js';
```

Comment:
Because this is a `commandsModule`, it should have access to the `commandsManager`. It's provided to `getCommandsModule` as an argument.